src/client/Microsoft.Identity.Client/ConfidentialClientApplication.cs

```diff
@@ -77,6 +77,35 @@ public sealed partial class ConfidentialClientApplication
                 authorizationCode);
         }
 
+        /// <summary>
+        /// Acquires a security token from the authority configured in the app using the authorization code
+        /// previously received from the STS.
+        /// It uses the OAuth 2.0 authorization code flow (See https://aka.ms/msal-net-authorization-code).
+        /// It's usually used in web apps (for instance ASP.NET / ASP.NET Core web apps) which sign-in users,
+        /// and can request an authorization code.
+        /// This method does not lookup the token cache, but stores the result in it, so it can be looked up
+        /// using other methods such as <see cref="IClientApplicationBase.AcquireTokenSilent(IEnumerable{string}, IAccount)"/>.
+        /// </summary>
+        /// <param name="scopes">Scopes requested to access a protected API</param>
+        /// <param name="authorizationCode">The authorization code received from the service authorization endpoint.</param>
+        /// <param name="codeVerifier">A dynamically created cryptographically random key used to provide proof of possession for the <paramref name="authorizationCode"/>.</param>
```

Comment:
Comments need to clearly state this is PKCE and add a link to it https://oauth.net/2/pkce/